libraries/chain/include/eosio/chain/transaction_context.hpp

```diff
@@ -83,6 +83,8 @@ namespace eosio { namespace chain {
 
          void validate_referenced_accounts( const transaction& trx, bool enforce_actor_whitelist_blacklist )const;
 
+         const bool is_dry_run()const { return trx_type == transaction_metadata::trx_type::dry_run; };
```

Comment:
Is there a reason to return a `const bool` instead of just `bool`?

Reply:
Thanks for your point. As it returns by a value of a built-in type, no need for `const`. Will remove it.